### PR TITLE
Refactor LoginActivity.java

### PR DESCRIPTION
**Pull request by jashasweejena**
Refactor all m* variable names to remove the 'm'
Used this to refer to the variable depending on its use in the code.

### CODE REVIEW

**Comment by jashasweejena** [comment]
[Tests-passed](https://github.com/jashasweejena/random_uploads/raw/master/Bunk-tests.png)

All tests passed too. 

Kindly merge 😄

File: app/src/main/java/app/abhijit/iter/LoginActivity.java [hunk 1]
```diff
@@ -37,6 +37,7 @@
 import android.support.v7.widget.Toolbar;
 import android.text.Editable;
 import android.text.TextWatcher;
+import android.util.Log;
```
Comment (abhijitparida) [inline app/src/main/java/app/abhijit/iter/LoginActivity.java]: Can you please remove this import?

Comment (jashasweejena) [reply app/src/main/java/app/abhijit/iter/LoginActivity.java]: sure
Can I change something in this PR? Or do I need to make another one?

Comment (abhijitparida) [reply app/src/main/java/app/abhijit/iter/LoginActivity.java]: Just make another commit and push it.

Comment (jashasweejena) [reply app/src/main/java/app/abhijit/iter/LoginActivity.java]: > Just make another commit and push it.

okay

**Comment by abhijitparida** [comment]
Issue #13 fixed.

**Pull request merged**